### DISCUSSION TITLE
What should I set for the cutoff when I use DeepPot-SE?

### DISCUSSION THREAD

**Discussion by captaingjq** [deepmodeling/deepmd-kit#16] [Q&A]
In the paper "Supplementary Materials for End-to-end Symmetry Preserving Inter-atomic Potential Energy Model for Finite and Extended Systems", there are many details of training a Mol. uni. model. I am trying to repeat this training for one of my class project. But the resulting error is huge(around 0.3eV after 80,000 training steps).
[mol_smth.json.txt](https://github.com/deepmodeling/deepmd-kit/files/2772076/mol_smth.json.txt)

The cutoff value must be provided in the json input file, I just set rcut_smth and r_cut to be very large. What value should I use?
I have 70,000 training data(10,000 for each molecule) including 'coord.raw', 'energy.raw', 'force.raw', 'type.raw'. But the program need 'box.raw', so I made a large 'box.raw' file where each row is same as '50 0 0 0 50 0 0 0 50'. Am I doing right by doing so?
I have attached the json file I used.

**Comment by jameswind** [comment]
Your input file looks good. The problem might be related to the data. How
do you convert the data? Is the energy in unit of eV and force eV/angstrom?

**Comment by captaingjq** [comment]
I use 'raw_to_set.sh' to convert the data. Yes, the energy in unit of eV and force eV/angstrom. Here is the raw data description:
"Suppose the molecule that we consider has N atoms. The type.dat file contains one line with the types of the N atoms written one by one. The atomic types are integers. Here we use 0, 1, 2, 3 for C, H, O, N, respectively. The coord.dat file contains 10000 lines. Each line provides all the 3 coordinate components of N atoms in 1 frame. The first three numbers are the 3 coordinate components of the first atom, while the second three numbers are the 3 coordinate components of the second atom. The force file force.dat is organized similarly to the coordinate file. The energy file energy.dat contains 10000 lines. Each line provides a number, namely the energy of the corresponding structure.
For each molecule, there are 1000 testing structures in the test folder. The format is the same with the
training structures, but the energies and forces are not given."

**Comment by jameswind** [comment]
I feel that the total training step is too small. Could you follow the Supp
to reproduce it?
In addition, an initial learning rate of 5e-3 might be too large for these
systems. Our experiences suggest that 5e-4 should be better. Please also
see the Supp.

**Comment by captaingjq** [comment]
I am not sure, 80,000 steps took me about 13 hours on my laptop. I could not imagine how long 2e+7 steps will take...Anyway, I will try to follow the Supp. By the way, about some parameters, like the batch_size, rcut_smth, r_cut, sel_a, the Supp didn't mention. What value should I use?

**Comment by captaingjq** [comment]
I have another question. Does 'raw_to_set.sh' change the ordering of data?

**Comment by captaingjq** [comment]
I finally found the problem of huge error. It's because 'raw_to_set.sh' changes the ordering of my predicted energies. Thank you very much for checking my input json file!

**Comment by GitWhilebear** [comment]
hi,
I can find the paper "Supplementary Materials for End-to-end Symmetry Preserving Inter-atomic Potential Energy Model for Finite and Extended Systems", can you give me a link?
